packages/xs-extra/forkexec.master/url

```diff
@@ -0,0 +1 @@
+archive: "https://github.com/xapi-project/forkexecd/archive/master.tar.gz"
```

Comment:
Checksum?

Reply:
Not for the ones tracking master, they are not versioned, it would change at each merge